### PR TITLE
Update plan-for-administrative-and-service-accounts.md

### PR DESCRIPTION
**Pull request by tseward**
Clarify #1734

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/security-for-sharepoint-server/plan-for-administrative-and-service-accounts.md [hunk 1]
```diff
@@ -190,11 +190,11 @@ For more information about service application endpoints, see [Using Service End
 |Distributed Cache   |X   |X   |
    
 > [!NOTE]
-> Some of the features that use the Distributed Cache service include:Newsfeeds, Authentication, OneNote client access, Security Trimming, and improves Page load performance. At least one Distributed Cache server is required in the farm.
+> Some of the features that use the Distributed Cache service are Newsfeeds, Authentication, OneNote client access, Security Trimming, and improves Page load performance. At least one Distributed Cache server is required in the farm.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/plan-for-administrative-and-service-accounts.md]:
```suggestion
> Some of the features that use the Distributed Cache service are Newsfeeds, Authentication, OneNote client access, Security Trimming, and improve Page load performance. At least one Distributed Cache server is required in the farm.
```

Comment (tseward) [reply SharePoint/SharePointServer/security-for-sharepoint-server/plan-for-administrative-and-service-accounts.md]: The plural of improve would be proper in this case.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**